packages/react-components/react-tag-picker-preview/src/components/TagPickerInput/useTagPickerInputStyles.styles.ts

```diff
@@ -130,7 +130,7 @@ export const useTagPickerInputStyles_unstable = (state: TagPickerInputState): Ta
       tagPickerInputClassNames.expandIcon,
       iconStyles.icon,
       iconStyles[state.size],
-      // state.disabled && iconStyles.disabled,
+      state.disabled && iconStyles.disabled,
```

Comment:
shouldn't the TagPickerControl also have some kind of disabled styles? 🤔 The background should be greyed out like the input?
![image](https://github.com/microsoft/fluentui/assets/20744592/1f1b1399-3d01-46fb-a55b-62e9d2113145)

vs

![image](https://github.com/microsoft/fluentui/assets/20744592/4803cf19-04df-4df5-af55-c392ff57c362)